packages/ui/src/styles/typography/typography.scss

```diff
@@ -4,10 +4,6 @@
 // TODO: Move it to GLOBAL-CSS service, so it can be turned off and scoped.
 @include va-typography;
 
-* {
```

Comment:
We need this in global typography styles, so it applies to all texts in p, etc. Maybe we don't need this, but it is useful when you just installed vuestic-ui in your project.